core/src/main/java/be/isach/ultracosmetics/UltraCosmeticsData.java

```diff
@@ -109,7 +109,7 @@ protected void checkTreasureChests() {
 
     protected boolean initModule() {
         SmartLogger logger = ultraCosmetics.getSmartLogger();
-        logger.write("Initializing module " + serverVersion);
+        logger.write("Initializing module " + serverVersion + " (server: " + UltraCosmeticsData.get().getServerVersion().getName() + ")");
```

Comment:
```suggestion
        logger.write("Initializing module " + serverVersion + " (server: " + serverVersion.getName() + ")");
```
We can shorten this because `serverVersion` here is actually the same variable that `getServerVersion()` is referencing.

Other than that, looks great!

Reply:
Alright, done :)